docs/installation.md

```diff
@@ -2,30 +2,107 @@
+  This resource defines OS deployment details for any machine attempting to register. The machine
+  registration is the entrance for Elemental nodes as it handles the authentication (based on TPM),
+  the Elemental Teal deployment and the node inclusion into to the MachineInventory so it can be added
+  to a cluster when there is a match based on a MachineInventorySelectorTemplate. The MachineRegistration
+  object includes the machine registration URL that nodes use to register against it.
+
+A Rancher Cluster resource is also required to deploy Elemental, it can be manually created as exemplified in
+the [Quick Start](quickstart.md) guide or created from the Rancher 2.6 UI.
+
+
+## Perpare Installation Media
+
+The installation media is the media that will be used to kick start an Elemental Teal deployment. Currently
+the supported media is a live ISO. The live ISO must include the registration configuration yaml hence it must
+crafted once the MachineRegistration is created. The installation media is created by using the `elemental-iso-build`
+helper script (see [quick start](quickstart.md#preparing-the-iso) guide)
+or by using the `elemental build-iso` command line utility included as part of the
+[Elemental Toolkit]({{elemental.toolkit.url}}/docs/creating-derivatives/build_iso).
+
+Within MachineRegistration only a subset of OS installation parameters can be configured, all available parameters listed
+at MachineRegistration reference page.
 
-## Elemental Partition Table
+In order to configure the installation beyond the common options provided within the
+[`elemental.install`](machineregistration-reference.md#configelementalinstall) section a `/etc/elemental/config.yaml`
+config file can be included into the squashed rootfs of the installation media. Note any configuration applied as
+part of `elemental.install` section of the MachineRegistration will be applied on top of the
+settings included in `/etc/elemental/config.yaml`.
 
-Elemental requires the following partitions.  These partitions are required by [{{elemental.toolkit.name}}]({{elemental.toolkit.url}})
+Most likely the cloud-init configuration is enough to configure and set the deployed node at boot, however
+if for some reason some sort of firstboot actions or scripts are required it is possible to also include
+Rancher System Agent plans into the installation media. Refer to the [Elemental Plans](elemental-plans.md) section for details for
+some example plans. The plans could be included into the squashed rootfs at `/var/lib/elemental/agent/plans`
+folder and they would be seen by the system agent at firstboot.
+
+
+## Start Installation Process
+
+The installation starts by booting the installation media on a node. Once the installation media has booted it will
+attempt to contact the management cluster and register to it by calling `elemental-register` command.
+For unattended installation the registration yaml configuration should have been already included as
```

Comment:
Do we now support any other that unattended installation? I thought that was deprecated, or at least there is no docs for that.

Reply:
Good point the text is missleading here. When I say unattended I meant completely automatic, you just need to press the power button. By interactive I was thinking on calling `elemental-register` manually. Gonna rewrite this part.